packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -107,6 +107,11 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
    */
   menuProps?: IContextualMenuProps;
 
+  /**
+   * Callback for when the Button's contextualmenu was closed (removed from the DOM)
+   */
+  onMenuDismissed?: () => void;
+
```

Comment:
Should we just be using the onMenuDismissed that could already be coming as part of the menuProps?